proposals/devcontainer-distribution.md

```diff
@@ -0,0 +1,71 @@
+# Community creation of templates and features and their distribution.
+
+This proposal is created to arrive at a schema where different communities and organizations can create features and devcontainer templates and make them discoverable to users.
+
+The proposal centers on the following ideas:
+
+- Discoverability for users both on the web (github.com) and on any tools that implement the [Dev container specification](../docs/specs/devcontainer-reference.md).
+- Users can validate the origin of the template or feature that they want to use.
+- Users can pin a particular version of a feature to allow consistent repeatable environments.
```

Comment:
```suggestion
- Users can pin a particular version of a feature to allow consistent, repeatable environments.
```

Reply:
Would users also pin versions of other tools included in their dev containers that aren't provided through a feature?